atomic_reactor/utils/remote_host.py

```diff
@@ -23,9 +23,9 @@
 SSH_COMMAND_TIMEOUT = 30
 SLOTS_RELATIVE_PATH = "osbs_slots"
 RETRY_ON_SSH_EXCEPTIONS = (paramiko.ssh_exception.NoValidConnectionsError,
```

Comment:
I would maybe rename this, since it is not only SSH exceptions

Reply:
but it is only for SSHRetrySession